README.md

```diff
@@ -208,15 +211,21 @@ iOS13 and higher
 iOS12 and lower
 * NSBluetoothPeripheralUsageDescription
 
-#### How to adjust ProGuard (Android)
+#### How to adjust ProGuard (Android)?
 
 In case you are using ProGuard add the following snippet to your `proguard-rules.pro` file:
 
 ```
 -keep class com.signify.hue.** { *; }
 ```
 
-This will prevent issues like [#131](https://github.com/PhilipsHue/flutter_reactive_ble/issues/131) .
+This will prevent issues like [#131](https://github.com/PhilipsHue/flutter_reactive_ble/issues/131).
+
+#### Why doesn't the BLE stack directly connect to my peripheral?
+
+Before you are able to execute BLE operations the ble stack of the device makes sure everything is setup correctly and then reports ready for duty. For some devices this takes a bit longer then for others. When starting the app make sure that the BLE-stack is properly intialized before you execute BLE operations. The safest way to do this is by listening to the `statusStream` and wait for `BleStatus.ready`.
```

Comment:
Always capitalize "BLE" (because it's an abbreviation).

Use a consistent spelling: either "BLE stack" (my preference) or "BLE-stack", not both.

"ready for duty" is probably fine, but I'd write "ready for operation".